libp2p/NodeTable.cpp

```diff
@@ -512,6 +510,10 @@ void NodeTable::onPacketReceived(
                         sourceNodeEntry = it->second;
                         sourceNodeEntry->lastPongReceivedTime =
                             RLPXDatagramFace::secondsSinceEpoch();
+
```

Comment:
I know this is good practice but do you know if we ever actually hit this case? It seems like we wouldn't given that pongs have to be received within 1 minute of the ping being sent to be considered valid?

Reply:
Not sure how it is related to Pong timeout, this is the case when there is a record in the bucket of the node table (maybe added long ago), but we receive a valid Pong with the same NodeID but from another endpoint.